### ISSUE TITLE
Redesigned search is showing wrong result summary

### ISSUE THREAD

**Issue by LukasReschke** [owncloud/core#13246]
When searching for something where only one result is shown a space is missing in the description: "1 Ordnerstimmt mit '<h1' überein"
![screen shot 2015-01-10 at 19 06 25](https://cloud.githubusercontent.com/assets/878997/5692584/cbae7306-98fb-11e4-9a92-85e15eebca23.png)

@butonic

**Comment by karlitschek** [comment]
@butonic can you have a look?

**Comment by MorrisJobke** [comment]
This is a problem of the translation. In the english original it is

```
n('files', ' matches \'{filter}\'', ' match \'{filter}\'', this.summary.totalDirs + this.summary.totalFiles, {filter: this.summary.filter})
```

And the German (and `cs_CZ`, `ru`, `es`, `it` and `nl`) misses that leading space. It's maybe better to add that space afterwards.

**Comment by MorrisJobke** [comment]
It's in `apps/files/js/filesummary.js` line 185 and 200.

**Comment by LukasReschke** [comment]
Mhm - sounds like the best idea would be to remove the spaces from the translation and add them dynamically via string concatenation or so.

**Comment by MorrisJobke** [comment]
I just noticed 2 translations that are correct and 6 incorrect ones.

**Comment by wakeup** [comment]
fixed with 2df331b6ed0909c018de8e686cae377c966454fb
